graphql.go

```diff
@@ -84,6 +83,13 @@ func UseStringDescriptions() SchemaOpt {
 	}
 }
 
+// Specifies whether to use struct field resolvers
```

Comment:
Comments need to start with the func/method name
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Please, use:
```
UseFieldResolvers...
```